app/assets/javascripts/modules/ajax_upload.js.coffee

```diff
@@ -13,6 +13,7 @@ class app.AjaxUpload
     form = $(input).closest('form')
     new app.Spinner().show(form)
     form.submit()
+    $(input).closest('form').reset()
```

Comment:
Yeah, should. That's the fun/weird part about this. I would really like an explanation for this behaviour. I tried `form.reset` multiple times, expecting different results. 

The current version works, I just cannot explain why `form.reset` does not work. I seems like the form is doing weird things in chrome.

A full explanation why this happens (also, which this is needed in Chrome in the first place) would be highly appreciated.